### PR TITLE
add KubeArmor ARM64 support

### PR DESCRIPTION
**Pull request by Ankurk99**
ref: #614 
Tested on Raspberry Pi 4
Kernel: Linux 5.15.0-1005-raspi
Distribution: Ubuntu 22.04 LTS

- [x] Support enforcement
- [x] Support observability
	- [x] accept and connect syscall
	- [x] support other syscalls
		- [x] support basic syscalls on arm (manually updated the syscall event ids)
		- [x] automatically detect the arch and compile for suitable architecture
- [x] fix failing ci
- [x] create arm based KubeArmor image(systemd)
- [x] create arm based KubeArmor image (docker image)
- [ ] update setup script to install suitable version of tools based on architecture

### CODE REVIEW

File: KubeArmor/BPF/Makefile [hunk 1]
```diff
@@ -13,6 +13,22 @@ endif
...
+UNAME_M := $(shell uname -m)
+UNAME_R := $(shell uname -r)
+
+ifeq ($(UNAME_M),x86_64)
+   ARCH = x86_64
+   LINUX_ARCH = x86
+   GO_ARCH = amd64
+endif
+
+ifeq ($(UNAME_M),aarch64)
```
Comment (nyrahul) [inline KubeArmor/BPF/Makefile]:
```suggestion
elsif ($(UNAME_M),aarch64)
```

**Comment by Ankurk99** [comment]
CI failing issue will possibly be fixed by https://github.com/kubearmor/KubeArmor/pull/778

**Comment by Ankurk99** [comment]
> CI failing issue will possibly be fixed by #778

The CI was randomly failing, re-ran the test and it just passed.

**Review by achrefbensaad — DISMISSED** [review]
seeing this error when running kubearmor on a k8s cluster (arm based) 
image

**Review by nyrahul — APPROVED** [review]
LGTM 👍

**Pull request merged**